### DISCUSSION TITLE
Strange things with abort.

### DISCUSSION THREAD

**Discussion by Zombie1995** [luc-github/ESP3D#603] [F.A.Q] [answered]
One more question. When i click abort button in webUI, printer does some strange work. It's stucking and doesn't react on any esp command, just rebooting helps. Marlin 2, esp3d 2.1.1, anycubic i3 mega.

**Comment by luc-github** [comment, marked as answer]
yes abort is emergency stop M112 https://marlinfw.org/docs/gcode/M112.html

**Reply by Zombie1995** [reply]
Are there any normal way for correct stopping printing?

**Reply by luc-github** [reply]
On Marlin only Print from SD command is available but none if printing from Host https://marlinfw.org/docs/gcode/M524.html

**Reply by Zombie1995** [reply]
Thanks again for your response ; )
